src/api/mod.rs

```diff
@@ -0,0 +1,288 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+mod error;
+mod responses;
+mod routes;
+
+use std::{
+    task::Poll,
+    time::SystemTime,
+};
+
+use actix::prelude::*;
```

Comment:
Would it be possible to import the required stuff explicitly, or is it just too much? I'm not a big fan of preludes.